main.tf

```diff
@@ -34,6 +40,12 @@ resource "azurerm_resource_group" "rg2" {
   tags = "${merge(var.common_tags,
     map("lastUpdated", "${timestamp()}")
     )}"
+    
+  # On creation, resource group is not ready without delay.
+  provisioner "local-exec" {
```

Comment:
Pretty sure this resource group isn't even used? so I would remove this change

Reply:
I had this thought also. 

Remove